requirements.txt

```diff
@@ -1,4 +1,5 @@
 click
 boto3
 pyyaml
-awscli
\ No newline at end of file
+awscli
+git+ssh://git@github.com/heetch/super-mario-python.git@v1.0.5
```

Comment:
This project is an open source project, intended to be used by us and the community in general. It shouldn't depend on private dependencies only us have access to.

Reply:
yup, same as above - didn't know it's open source :)